share/completions/zef.fish

```diff
@@ -0,0 +1,94 @@
+complete -c zef -f
+complete -c zef -n __fish_is_first_token -s h -l help -f -d 'Display help text'
+complete -c zef -n __fish_is_first_token -l version -f -d 'Detailed version information'
+complete -c zef -n 'not __fish_is_first_token' -l config-path -d 'Load a specific Zef config file'
+complete -c zef -n '__fish_seen_argument -l config-path' -x -k -a '(__fish_complete_suffix)'
+complete -c zef -n 'not __fish_is_first_token' -l error -d 'Verbosity level'
+complete -c zef -n 'not __fish_is_first_token' -l warn -d 'Verbosity level'
+complete -c zef -n 'not __fish_is_first_token' -l info -d 'Verbosity level (default)'
+complete -c zef -n 'not __fish_is_first_token' -s v -l verbose -d 'Verbosity level'
+complete -c zef -n 'not __fish_is_first_token' -l debug -d 'Verbosity level'
+
+# Commands
+complete -c zef -n __fish_use_subcommand -a install -d 'Install specific distributions by name or path'
+complete -c zef -n __fish_use_subcommand -a uninstall -f -d 'Uninstall specific distributions'
+complete -c zef -n __fish_use_subcommand -a test -d 'Run tests on a given module\'s path'
+complete -c zef -n __fish_use_subcommand -a fetch -d 'Fetch and extract module\'s source'
+complete -c zef -n __fish_use_subcommand -a build -d 'Run the Build.pm in a given module\'s path'
+complete -c zef -n __fish_use_subcommand -a look -x -d 'Download a single module and change into its directory'
+complete -c zef -n __fish_use_subcommand -a update -d 'Update package indexes for repositories'
+complete -c zef -n __fish_use_subcommand -a upgrade -d 'Upgrade specific distributions (or all if no arguments)'
+complete -c zef -n __fish_use_subcommand -a search -d 'Show a list of possible distribution candidates for the given terms'
+complete -c zef -n __fish_use_subcommand -a info -d 'Show detailed distribution information'
+complete -c zef -n __fish_use_subcommand -a browse -d 'Browse a distribution\'s available support urls (homepage, bugtracker, source)'
+complete -c zef -n __fish_use_subcommand -a list -d 'List known available distributions, or installed distributions with `--installed`'
+complete -c zef -n __fish_use_subcommand -a depends -d 'List all direct and transitive dependencies for a given identity'
```

Comment:
"Direct and transitive" can just be implied.

"List dependencies for a given identity"?

Reply:
Or "List *full* dependencies for a given identity"?